### ISSUE TITLE
Instructions for submitting the ASAP and final PDFs

### ISSUE THREAD

**Issue by dwsideriusNIST** [livecomsjournal/livecomsjournal.github.io#170]
@oostenbrink and I ran into a question while completing the Sampling-Uncertainty paper, post-acceptance.

Is it necessary to upload a PDF to the ScholasticaHQ system somewhere? Neither Chris nor I could find a link or place within HQ to upload the ASAP (and, later, the final) PDF. In the interim, I just sent the ASAP PDF to Chris and he's handling it from there.

We can update the author and editor instructions once this step is better understood.

**Comment by davidlmobley** [comment]
No, we (@dmzuckerman @mrshirts and myself) are discussing this. We'll probably make the policy that we (the managing editors) will just pull a final PDF from your GitHub and upload it ourselves; there doesn't seem to be an easy way to let you do it. :(

**Comment by mrshirts** [comment]
Addressing this in #171

**Comment by dwsideriusNIST** [comment]
I take it that we have DOIs now?

https://doi.org/10.33011/livecoms.1.1.5067

**Comment by mrshirts** [comment]
Yes!
